app/views/roles/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for(:title, "Roles") %>
+
+<% if @roles.empty? %>
+  <div class="none-found">
+    No roles found.
+  </div>
+<% else %>
+  <% @roles.each do |role| %>
+    <h3><%= role.name.humanize %></h3>
+    <div id="directory-album" class="flowing-columns-max-4 hidden-print">
+      <% role.users.each do |user| %>
+        <% user = user.decorate %>
+        <div class="user section clearfix <%= user.active? ? "" : "inactive" %>">
+          <div class="photo-wrapper">
+            <%= link_to(user.photo_if_permitted(:thumb), user) %>
+          </div>
+          <div class="info">
+            <% if user.preferred_contact.present? %>
+              <%= icon_tag(user.preferred_contact_icon,
+                class: "preferred-contact", title: user.preferred_contact_tooltip) %>
+            <% end %>
+            <div><%= link_to(user.name, user) %></div>
+            <div class="email"><%= user.email_link if user.email %></div>
+            <div class="unit-phone"><%= user.unit_and_phone %></div>
+          </div>
+        </div>
```

Comment:
For this area, I think just a list of names under each role should be sufficient. See the UserDecorator class, which provides a method to create a user link.